lib/Tool/Session.php

```diff
@@ -29,83 +30,165 @@ final class Session
      */
     private static $handler;
 
-    public static function getHandler(): AdminSessionHandlerInterface
+    /**
+     * @desc This is for forward compatibility, the custom Session implementation is not being used anymore in Pimcore 11.
+     * @desc For forward compatibility, you can use this class and pass the SessionInterface from the request, in Pimcore 10.6, the Admin Session will be used instead
+     *
+     * @param SessionInterface $session Session being used is the Admin Session, Varaible is not used here since the dedicated Admin Session is used. Please pass the Request SessionInterface here for forward compatibility
+     *
+     * @param callable(AttributeBagInterface, SessionInterface):mixed $func
+     *
+     */
+    public static function useBag(SessionInterface $session, callable $func, string $namespace = 'pimcore_admin'): mixed
+    {
+        return self::getSessionHandler()->useSessionAttributeBag($func, $namespace);
+    }
+
+    /**
+     * @desc This is for forward compatibility, the custom Session implementation is not being used anymore in Pimcore 11.
+     * @desc For forward compatibility, you can use this class and pass the SessionInterface from the request, in Pimcore 10.6, the Admin Session will be used instead
+     *
+     * @param SessionInterface $session Session being used is the Admin Session, you can safely pass nothing here
```

Comment:
```suggestion
     * @param SessionInterface $session Parameter is not used here since the dedicated Admin Session is used. Please pass the Request SessionInterface here for forward compatibility
```